tests/helpers.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	"context"
+)
+
+// VeryVerbose when set to true, will enable very verbose logging of services. Controlled via
+// flag in tests_test package.
+var VeryVerbose bool
```

Comment:
I don't see any spots where this is assigned

Reply:
It is intended for engineers, if they wish to increase verbosity.

I've changed the comment to:

```go
// VeryVerbose when set to true, will enable very verbose logging of services.
```